configs/default/CUST-SIRMIXALOT.config

```diff
@@ -0,0 +1,187 @@
+# 
+
+# diff all bare
+
+# version
+# Betaflight / STM32F7X2 (S7X2) 4.1.0 Sep 25 2019 / 14:09:57 (08e8afa09) MSP API: 1.42
+# manufacturer_id: CUST   board_name: SIRMIXALOT   custom defaults: NO
+
+board_name SIRMIXALOT
+manufacturer_id CUST
+
+# resources
+resource BEEPER 1 C11
+resource MOTOR 1 B06
+resource MOTOR 2 B07
+resource MOTOR 3 B08
+resource MOTOR 4 B09
+resource LED_STRIP 1 B01
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_TX 11 A00
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource SERIAL_RX 11 B11
+resource I2C_SCL 1 A08
+resource I2C_SDA 1 C09
+resource LED 1 B12
+resource LED 2 C08
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource CAMERA_CONTROL 1 C10
+resource ADC_BATT 1 C02
+resource ADC_RSSI 1 C03
+resource ADC_CURR 1 C01
+resource BARO_CS 1 A15
+resource FLASH_CS 1 C00
+resource OSD_CS 1 C14
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 B02
+
+# timer
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer B09 AF2
+# pin B09: TIM4 CH4 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature SOFTSERIAL
+feature TELEMETRY
+feature LED_STRIP
+feature OSD
+
+# beeper
+beeper -ON_USB
+
+# serial
+serial 2 2048 115200 57600 0 115200
+serial 3 1024 115200 57600 0 115200
+serial 4 64 115200 57600 0 115200
+serial 30 16384 115200 57600 0 115200
+
+# led
+led 0 7,7::A:2
+
+# color
+color 10 240,128,255
+
+# mode_color
+mode_color 6 1 2
+
+# aux
+aux 0 0 0 1700 2100 0 0
+aux 1 27 8 1300 2100 0 0
+aux 2 13 4 1300 2100 0 0
+aux 3 15 5 1300 2100 0 0
+aux 4 26 0 900 2100 0 0
+aux 5 33 2 1300 1700 0 0
+aux 6 34 3 1300 1700 0 0
+aux 7 36 6 1700 2100 0 0
+
+# vtxtable
+vtxtable bands 5
+vtxtable channels 8
+vtxtable band 1 BANDA    A FACTORY 5865 5845 5825 5805 5785 5765 5745 5725
+vtxtable band 2 BANDB    B FACTORY 5733 5752 5771 5790 5809 5828 5847 5866
+vtxtable band 3 BANDE    E FACTORY 5705 5685 5665 5645 5885 5905 5925 5945
+vtxtable band 4 BANDF    F FACTORY 5740 5760 5780 5800 5820 5840 5860 5880
+vtxtable band 5 RACE     R FACTORY 5658 5695 5732 5769 5806 5843 5880 5917
+vtxtable powerlevels 4
+vtxtable powervalues 25 200 500 800
+vtxtable powerlabels 25 200 500 800
+
+# master
+set acc_calibration = 24,-24,38
```

Comment:
This should not be in here, as it is specific to the acc chip on one board.

Reply:
Again, used diff command and it splatted that out...I'll remove it.

Reply:
It looks like you forgot to reset your custom target to defaults in step 1 (https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md#1-flash-the-firmware-onto-your-board):

- verify that your board is properly reset to `defaults`: The output of a `diff hardware` in CLI should show `board_name`, `manufacturer_id`, and lines starting with a `#`.